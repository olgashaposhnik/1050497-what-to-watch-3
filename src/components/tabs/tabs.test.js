import React from "react";
import renderer from "react-test-renderer";
import Tab from '../tab/tab.jsx';
import Tabs from '../tabs/tabs.jsx';

const TAB_NAMES = [`TabName1`, `TabName2`, `TabName3`];

it(`Tabs should render correctly`, () => {
  const wrapper = renderer
    .create(
        <Tabs>
          <Tab name={TAB_NAMES[0]}>TabName1</Tab>
          <Tab name={TAB_NAMES[1]}>TabName2</Tab>
          <Tab name={TAB_NAMES[2]}>TabName3</Tab>
        </Tabs>
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
