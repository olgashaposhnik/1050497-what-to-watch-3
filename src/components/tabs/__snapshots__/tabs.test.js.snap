// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should render correctly 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          TabName1
        </a>
      </li>
      <li
        className="movie-nav__item "
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          TabName2
        </a>
      </li>
      <li
        className="movie-nav__item "
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          TabName3
        </a>
      </li>
    </ul>
  </nav>
  TabName1
</div>
`;
